docs/add-ons/testing.md

```diff
@@ -5,11 +5,12 @@ title: "Local add-on testing"
 The fastest and recommended way to develop add-ons is using a local Visual Studio Code dev environment. The [Official Add-ons][hassio-addons] repository includes a dev container setup for VS Code which will run Supervisor and Home Assistant, with all of the add-ons mapped as Local Add-ons inside, making it simple for add-on developers on Windows, Mac and Linux desktop OS-es. 
 
 - Follow the instructions to download and install the [Remote Containers][remote-containers] VS Code extension.
-- Copy the `.devcontainer` and `.vscode` folder from [Official Add-ons][hassio-addons] repository into the root of your add-ons folders.
-- Open the root folder inside VS Code, and when prompted re-open the window inside the container (or, from the Command Palette, select 'Rebuild and Reopen in Container'). 
+- Create a folder called "addons"
```

Comment:
It is "correct", this is documenting a workaround for a "bug", the script expects the root dir to be named "addons", which should be adjusted in the script instead of documented like this.

Reply:
<https://github.com/home-assistant/addons/pull/2000>